lib/pubnub/client/getters_setters.rb

```diff
@@ -54,9 +54,9 @@ def change_user_id(user_id)
         @env[:user_id] = user_id
       end
 
-      alias session_uuid= change_user_id
-      alias uuid= change_user_id
-      alias set_uuid= change_user_id
+      alias session_uuid= change_uuid
+      alias uuid= change_uuid
+      alias set_uuid= change_uuid
```

Comment:
@kleewho actually `change_user_id` was also goo enough. With `change_uuid` here it will just additional method call.

Reply:
If I change it to `change_user_id` any client using it won't get deprecation information and we won't achieve the goal of changing the language from `uuid` to `user_id`